### DISCUSSION TITLE
How to add pageination Text

### DISCUSSION THREAD

**Discussion by goodlythink** [icflorescu/mantine-datatable#17] [Q&A]
is posible change pagination text

e.g.
1-15/500
to

item 1-15 from 500

**Comment by icflorescu** [comment]
Not at the moment, but obviously we should be able to do so.
Thanks for bringing this up.

I'm thinking about adding an optional property called `paginationText`, that could work like this:

```ts
<DataTable
  paginationText={({ from, to, total }) => `Item ${from}-${to} of ${total}`}
 // ...
/>
```

What do you think?

**Reply by icflorescu** [reply]
This should be fairly easy to implement and I think I could come up with a new version today.

**Reply by icflorescu** [reply]
Added [issue](https://github.com/icflorescu/mantine-datatable/issues/18).

**Reply by icflorescu** [reply]
Done. Starting with `v.1.1.2` you can use `paginationText` property.

**Reply by goodlythink** [reply]
nice. thank you

**Comment by icflorescu** [comment]
Starting with `v.1.1.2`, pagination text is customizable. See [icflorescu.github.io/mantine-datatable/examples/pagination](https://icflorescu.github.io/mantine-datatable/examples/pagination).
